lotto/src/main/kotlin/lotto/domain/Lotto/Numbers.kt

```diff
@@ -0,0 +1,14 @@
+package lotto.domain.Lotto
+
+class Numbers {
+
+    companion object {
+        private val values = mutableListOf<Number>().also {
+            for (i in 1..45)
+                it.add(Number(i))
+        }
```

Comment:
also 배워갑니다 👍

Reply:
다만 `(1..45)`로 그냥 사용해도 상관없을 것 같아요. 
```suggestion
        private val values = (1..45)
```
`(1..45)`로 사용하게 되면 IntRange 라는 타입으로 나오게 되는데, IntRange도 Iterable의 구현체이기 때문에 shuffled() 메서드가 구현되어 있어서 아래의 getShuffledNumbers에서 그대로 사용할 수 있을 것 같아요.

Reply:
ㄷㄷ 제가 생각하던것보다 훨씬 편하네요 감사합니다!